cmd/kubeadm/app/master/selfhosted.go

```diff
@@ -333,3 +358,175 @@ func getSchedulerDeployment(cfg *kubeadmapi.MasterConfiguration, volumes []v1.Vo
 func buildStaticManifestFilepath(name string) string {
 	return path.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, "manifests", name+".yaml")
 }
+
+func apiServerPKISecretVolume() v1.Volume {
+	return v1.Volume{
+		Name: "k8s-pki",
+		VolumeSource: v1.VolumeSource{
+			Secret: &v1.SecretVolumeSource{
+				SecretName: kubeadmconstants.PKISecretName,
```

Comment:
Is there a reason why we keep on embedding the yaml into the code like this @luxas?   Why can we use gobindata similar to the e2es and just reference the target names.  This way yamls are just loaded and built into the binary? 

/cc @luxas